app/web/markdown/blog/2024/05/20/introducing-strong-verification.md

```diff
@@ -0,0 +1,51 @@
+
+Your data will be securely stored on the server, with the most sensitive data encrypted and inaccessible to the server itself. You can read more about how we store and process your data [here](link).
+
+## Is Strong Verification free? Do I have to pay for it?
+
+**Every user will be able to complete Strong Verification for free until 1st October, 2024.**
+
+Strong Verification costs the Couchers, Inc. 501(c)3 non-profit real money. We do not currently have an abundance of funds, and so the board will have to make a decision regarding funding for this feature. Please [consider donating](https://couchers.org/donate).
+
+## What information is verified?
+
+We verify your date of birth and gender^[footnote: passports do not contain information about gender, only sex. We are forced to make a simplifying assumption regarding sex and gender. We also have a mechanism for those for whom this assumption does not hold, you can find out more [here](todo link).] match the details listed on your profile.
+
+## Why should I complete Strong Verification?
+
+By completing Strong Verification, other users will have the peace of mind that they are interacting with a real human. Strong Verification also stops people who were banned for violating the terms of service or community guidelines from creating new verified accounts^[footnote: Each passport can opnly be verified once, so users who are banned cannot re-verify using the same passport.]. Additionally, this data may be helpful if we ever need to collaborate with criminal investigations.
+
+Finally, users with Strong Verification will show up higher in search results and have a higher chance of both receiving requests and having their host requests accepted. In the coming weeks and months, we will be rolling out further features based on Strong Verification, providing further incentive to verify.
+
+## Is Strong Verification compulsory?
+
+No, we believe in giving all of our community members the ability to use the platform how they wish, as long as they follow the community guidelines and terms of service, and follow the spirit of our community. We know some members do not feel comfortable verifying their information using this mechanism, and that not all members are able to do verification, even if they would like to do so; either because they don't have a compatible passport or a compatible phone. Users who do not complete Strong Verification will still be able to use the full features of the platform, except those few features that build on top of Strong Verification. Strong Verificaiton increases visibility and trust, but those who do not wish to complete it will still be able to use Couchers to the full extent.
```

Comment:
No. We know some members do not feel comfortable verifying their information using this mechanism, and that not all members are able to do verification, even if they would like to do so; either because they don't have a compatible passport or a compatible phone.